components/camel-chatscript/src/main/java/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Class-Path: 
+

```

Comment:
Here you need to declare service stuff, take a look at the other components in the repo

Reply:
Hi, I checked the existing components and none of the components are having this file. I am planning to remove it. Please let me know if you have any concerns,

Reply:
Removed the file based  after reviewing other components